### PR TITLE
 French translation

### PR DESCRIPTION
**Pull request by MininuxDev**

### CODE REVIEW

File: locales/FR_fr.json [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+				"answer": "Malheuresement, non. Les NNIDs existants ne fonctionneront pas sur Pretendo car seul Nintendo détient vos données utilisateur. Bien qu'une migration de NNID à PNID soit théoriquement possible, elle serait risquée et nécessiterait des données personnelles sensibles que nous préférons ne pas stocker."
+
+			},
+			{
+				"question": "Comment puis-je utiliser Pretendo ?",
+				"answer": "Pretendo n'est actuellement pas dans un état prêt à être utilisé par le public. Cependant, dès qu'il le sera, vous pourrez utiliser Pretendo en exécutant simplement notre patcheur homebrew sur votre console."
+			},
+			{
+				"question": "Savez-vous quand <fonctionnalité/service> sera prêt ?",
+				"answer": "Non. De nombreuses fonctionnalités/services de Pretendo sont developpés indépendamment (par exemple, un développeur peut travailler sur Miiverse tandis qu'un autre travaille sur Comptes et Amis) et nous ne pouvons donc pas donner d'estimation exacte du temps qu'il faudra pour que tout soit prêt."
```
Comment (montylion) [inline locales/FR_fr.json:49]:
```suggestion
				"answer": "Non. De nombreuses fonctionnalités/services de Pretendo sont développés indépendamment (par exemple, un développeur peut travailler sur Miiverse tandis qu'un autre travaille sur Comptes et Amis) et nous ne pouvons donc pas donner d'estimation exacte du temps qu'il faudra pour que tout soit prêt."
```

File: locales/FR_fr.json [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+
+			},
+			{
+				"question": "Est-ce que Pretendo fonctionne sur Cemu/émulateurs ?",
+				"answer": "Pretendo est conçu pour la Wii U et la 3DS; pour le moment, le seul émulateur pour ces consoles qui supporte le NN est Cemu. Cemu ne prend pas officiellement en charge les serveurs personnalisés, mais il devrait toujours être possible d'utiliser Pretendo avec Cemu.<br>Pretendo ne prend pas en charge Cemu actuellement."
```
Comment (montylion) [inline locales/FR_fr.json:54]:
```suggestion
				"answer": "Pretendo est conçu pour la Wii U et la 3DS; pour le moment, le seul émulateur pour ces consoles qui supporte le NN est Cemu. Cemu ne prend pas officiellement en charge les serveurs personnalisés, mais il devrait être possible d'utiliser Pretendo avec Cemu.<br>Pretendo ne prend pas en charge Cemu actuellement."
```

File: locales/FR_fr.json [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+				"special": "Remerciement spéciaux",
+				"caption": "Étude des structures de données de Nintendo",
+				"picture": "https://cdn.discordapp.com/avatars/186572995848830987/b55c0d4e7bfd792edf0689f83a25d8ea.png?size=128",
+				"github": "https://github.com/Kinnay"
+			}
+		]
+	},
+	"progressPage": {
+		"title": "Notre progression",
+		"description": "Vérifiez l'avancement et les objectifs du projet ! (Mis à jour environ toute les heures, ne reflète pas TOUS nos objectifs et progression)."
```
Comment (montylion) [inline locales/FR_fr.json:139]:
```suggestion
		"description": "Vérifiez l'avancement et les objectifs du projet! (Mis à jour environ toute les heures, ne reflète pas TOUS nos objectifs et notre progression)."
```

**Review by montylion — CHANGES_REQUESTED** [review]
Could you check my changes?
I hope I did this right, it's the first time I review a pull request

File: locales/FR_fr.json (hunk 3 above)
Comment (MininuxDev) [reply locales/FR_fr.json:139]: According to this website https://grammaire.reverso.net/les-espaces-et-la-ponctuation/, there should always be a space before and after exclamation and questions marks

Comment (MininuxDev) [reply locales/FR_fr.json:139]: however the part about the missing "notre" seems right, i don't know if it's possible to partially commit a suggestion ?

Comment (montylion) [reply locales/FR_fr.json:139]: Thanks for pointing that out! Fixed the suggestion so that it only corrects notre.

**Comment by montylion** [comment]
Merged, thanks for contributing!

**Comment by ashmonty** [comment]
Hey @MininuxDev, I'm Ash from the Pretendo Network developer team.

I'm contacting you because you have contributed to our `PretendoNetwork/website` repository in the past, and we're working on relicensing it under the AGPL3 license (https://choosealicense.com/licenses/agpl-3.0/), which protects the code from being used in closed-source projects, while still permitting modification, as long as the source code is made available. To do so, we need to ask every contributor to also relicense their contributions to AGPL3.

Do you agree to relicense your contribution(s) under AGPL3? (Yes/No)

P.S: I'm asking you here because I wasn't sure where to reach you otherwise

**Comment by MininuxDev** [comment]
Hi @ashmonty , this is great news I say yes !

**Comment by ashmonty** [comment]
That's great, thank you very much!

**Pull request merged**